docs/src/common.md

```diff
@@ -233,18 +247,28 @@ Finally, we comment on the time spent by each integration.
 ```@example common
 @elapsed solve(prob, Vern9(), abstol=1e-20);
 ```
-Clearly, the integration with `TaylorMethod()` takes *much longer* than that using
+The integration with `TaylorMethod()` takes *much longer* than that using
 `Vern9()`. Yet, as shown above, the former preserves the Jacobi constant
-to a high accuracy while displaying the correct dynamics; whereas the latter
-solution loses accuracy in the sense of not conserving the Jacobi constant,
+to a high accuracy, whereas the latter
+solution looses accuracy in the sense of not conserving the Jacobi constant,
```

Comment:
Should it say "loses" with only one "o"?

Reply:
Good catch! Changed